src/components/GameCard.js

```diff
@@ -1,68 +1,132 @@
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
+import {useState} from 'react'
+import {ReactComponent as Bin} from '../assets/bin-icon.svg'
+import ButtonPrimary from './Buttons/ButtonPrimary'
+import ButtonSecondary from './Buttons/ButtonSecondary'
 
 GameCard.propTypes = {
     location: PropTypes.string.isRequired,
     date: PropTypes.string.isRequired,
     players: PropTypes.string.isRequired,
     winner: PropTypes.string.isRequired,
     shots: PropTypes.string.isRequired,
+    id: PropTypes.string.isRequired,
+    onDelete: PropTypes.func.isRequired,
 }
 
-export default function GameCard({location, date, players, winner, shots}) {
- return(
-    <Card>
-        <Location>{location}</Location>
-        <Date>{date}</Date>
-       
-            <Headline>Player(s)</Headline>
-            <DataPlayers>{players}</DataPlayers>
-      
-            <Headline>Winner(s)</Headline>
-            <DataWinner>{winner}</DataWinner>
-        
-       
-            <Headline>Total Shots</Headline>
-            <DataShots>{shots}</DataShots>
-    </Card>
- )
+export default function GameCard({location, date, players, winner, shots, id, onDelete}) {
+ 
+    const [isSetToDelete, setIsSetToDelete] = useState(false)
+
+    return(
+        <Card>
+            {!isSetToDelete && (
+                <SavedGameContent>
+                    <Location>{location}</Location>
+                    <Date>{date}</Date>
+                    <PlayerWrapper>
+                        <Headline>Player(s)</Headline>
+                        <span>{players}</span>
+                    </PlayerWrapper>
+                    <WinnerWrapper>
+                        <Headline>Winner(s)</Headline>
+                        <span>{winner}</span>
+                    </WinnerWrapper>
+                    <ShotsWrapper>
+                        <Headline>Total Shots</Headline>
+                        <span>{shots}</span>
+                    </ShotsWrapper>
+                    <ButtonDeleteFirst onClick={() => setIsSetToDelete(true)}><BinIcon/></ButtonDeleteFirst>
+                </SavedGameContent>
+            )}
+            {isSetToDelete && (
+                <DeleteField>
+                    <span>Do you want to delete this gamecard?</span>
+                    <ButtonDeleteSecond onClick={() => onDelete(id)}><BinIcon/>Delete</ButtonDeleteSecond>
```

Comment:
Perhaps you could display some of the original data from the card which is set to delete on the delete card. For example the date and/or the location. Just to make sure that I selected the correct card before I click the button.